{"name": "User cannot login with invalid ids and passwords -- @1.3 ", "status": "passed", "steps": [{"name": "Given Open the my account page", "status": "passed", "start": 1618434362486, "stop": 1618434365207}, {"name": "When User tries log in with test@test.com and test", "status": "passed", "start": 1618434365208, "stop": 1618434365961}, {"name": "Then Verify login was failed", "status": "passed", "start": 1618434365961, "stop": 1618434365971}, {"name": "Then Verify correct error message was displayed", "status": "passed", "start": 1618434365971, "stop": 1618434365987}], "parameters": [{"name": "email", "value": "test@test.com"}, {"name": "password", "value": "test"}], "start": 1618434360899, "stop": 1618434366051, "uuid": "1e0d0bfe-6895-4b28-b15e-7859cd1bf7a4", "historyId": "0cfaefdc300f5fbf85f165a5c21aa1b6", "fullName": "Test Scenarios for Account Login Error Handling in GetTop: User cannot login with invalid ids and passwords", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Scenarios for Account Login Error Handling in GetTop"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}